index.md

```diff
@@ -60,7 +60,7 @@ title: Sean Lo's Resume
 <span> Redlands, CA <span class="small-right">Jun. 2020 - Mar. 2021</span></span>
 
 * Oversaw end-to-end release and testing of flagship ArcGIS enterprise products across 6 web and mobile platforms with ArcGIS Dashboard reaching 1,000,000,000+ calls daily
-* Designed and maintained automation scripts for ArcGIS Dashboard and ArcGIS Insights using Python, Selenium, and Jenkins, ensuring efficient and reliable testing of these critical products
+* Designed and maintained automation scripts for ArcGIS Dashboard and ArcGIS Insights using Python, Selenium, and Jenkins, ensuring efficient and reliable testing of these critical products resulting in xyz
 * Developed and executed comprehensive test plans and test matrices as part of the product release cycle
 * Performed rigorous testing on Release Candidates to ensure the high quality and stability of product releases
 * Collaborated with Project Managers, Product Engineers and other teams to ensure successful release of products
```

Comment:
These last 3 bullets are very similar and redundant. Any way to rephrase or be more specific? Can likely remove at least one of them if you need the space